RevenueCatUI/CustomerCenter/Views/SubscriptionDetailView.swift

```diff
@@ -54,7 +54,7 @@ struct SubscriptionDetailView: View {
         customerInfoViewModel: CustomerCenterViewModel,
         screen: CustomerCenterConfigData.Screen,
         purchaseInformation: PurchaseInformation?,
-        virtualCurrencies: [String: VirtualCurrencyInfo]?,
+        virtualCurrencies: RevenueCat.VirtualCurrencies?,
```

Comment:
I see we're using `RevenueCat.` prefix for some types, was it required to disambiguate something?